File: cpp/game-server/tools/regscan/tests/fixtures/valid/handlers/aion/gameserver/handlers/quest/heiron/_1500OrdersFromPerento.cpp
#include "aion/gameserver/handlers/HandlerRegistry.h"
#include "regscan_fixture/FakeCore.h"

namespace aion::gameserver::handlers::quest::heiron {

class _1500OrdersFromPerento final : public questEngine::handlers::AbstractQuestHandler {
public:
	_1500OrdersFromPerento() : AbstractQuestHandler(1500) {}
};
AION_QUEST_HANDLER(_1500OrdersFromPerento, 1500);

} // namespace aion::gameserver::handlers::quest::heiron

File: cpp/game-server/tools/regscan/tests/fixtures/valid/java/handlers/quest/heiron/_1500OrdersFromPerento.java
package quest.heiron;

import com.aionemu.gameserver.questEngine.handlers.AbstractQuestHandler;

public class _1500OrdersFromPerento extends AbstractQuestHandler {

	public _1500OrdersFromPerento() {
		super(1500);
	}

	@Override
	public void register() {
		qe.registerQuestNpc(204500).addOnQuestStart(questId);
	}
}
